tests/integration/multipage/test_errors.py

```diff
@@ -0,0 +1,145 @@
+import unittest
+from app import create_app
+from tests.integration.create_token import create_token
+
+
```

Comment:
All integration tests now need to extend IntegrationTestCase as that setups up an in-memory sql database. otherwise the tests will just fail

from tests.integration.integration_test_case import IntegrationTestCase